puppet-lint-check_unsafe_interpolations.gemspec

```diff
@@ -0,0 +1,29 @@
+require_relative 'lib/puppet/lint/check_unsafe_interpolations/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "puppet-lint-check_unsafe_interpolations"
+  spec.version       = Puppet::Lint::CheckUnsafeInterpolations::VERSION
```

Comment:
We should implement this class in the PR. The version can be set to 0.0.1.

For the correct structure you can look at the example that I referenced yesterday.